packages/integrations/src/scrub.ts

```diff
@@ -0,0 +1,102 @@
+import { Event, EventHint, EventProcessor, Hub, Integration } from '@sentry/types';
+import { isPlainObject, isRegExp, normalize } from '@sentry/utils';
+
+/** JSDoc */
+interface ScrubOptions {
+  sanitizeKeys: Array<string | RegExp>;
+}
+
+/** JSDoc */
+export class Scrub implements Integration {
```

Comment:
I'd use a noun for the integration name.

```suggestion
export class Scrubber implements Integration {
```

Reply:
Thank you for the feedback! I will change later.

Reply:
I just pushed the changes to use noun. All the `yarn build`, `lint` and `test` are worked on my laptop. Thanks!

Reply:
Actually, other integrations use verbs (`RewriteFrames`, `Debug`, `Dedupe`, `CaptureConsole` - rather than `FrameRewriter`, `Deduper`, etc), so `Scrub` actually fits better in that pattern, though you might consider something like `ScrubData` or `SanitizeData`, since other integrations include the object of the verb in the name.

Reply:
Thank you for clearify naming convention. `ScrubData` is much more good name I think, therefore I renamed the name at latest commit.